Shopify/hydrogen
# Clean up package lock json

### WHY are these changes introduced?

* Re-generate package.json (1.6MB -> 765kB)
* Fix `react` version mismatch

### HOW to test your changes?

1. Remove your `node_modules` folder and run `npm install`
1. Search for `"name": "react"` in your node_modules, you should get exactly 1 instance of this search (Making sure peer dependency works)

#### Post-merge steps



#### Checklist
